scripts/combine_profiles.py

```diff
@@ -0,0 +1,348 @@
+        # Get current time
+        now = datetime.now()
+        revision['date'] = str(now.year) + '-' + str(now.month).zfill(2) + '-' + str(now.day).zfill(2)
+        revision['author'] = 'Merge tool'
+        revision['comment'] = 'Initial revision'
+        profiles[profile_name]['history'].append(revision)
+        profiles[profile_name]['capabilities'] = list()
+        profiles[profile_name]['capabilities'].append('baseline')
+        return profiles
+
+    def get_api_version(self, profile1, profile2):
+        api_version_str = profile1['api-version']
+        api_version_2_str = profile2['api-version']
+        api_version = self.get_api_version_list(api_version_str)
+        api_version_2 = self.get_api_version_list(api_version_2_str)
+        for i in range(len(api_version)):
+            if (api_version[i] > api_version_2[i]):
+                break
+            elif (api_version[i] < api_version_2[i]):
+                api_version_str = api_version_2_str
+                api_version = api_version_2
+                break
+        return [api_version_str, api_version]
+
+    def get_profile_description(self, profile_name1, profile_name2):
+        return 'Merged profile from ' + profile_name1 + ' and ' + profile_name2
+
+    def get_version_from_schema(self, schema):
+        needle = 'profiles-'
+        pos = schema.find(needle) + len(needle)
+        ver = schema[pos:]
+        version = list()
+        version.append(ver[:ver.find('.')])
+        ver = ver[ver.find('.') + 1:]
+        version.append(ver[:ver.find('.')])
+        ver = ver[ver.find('.') + 1:]
+        version.append(ver[:ver.find('-')])
+        ver = ver[ver.find('-') + 1:]
+        version.append(ver[:ver.find('.')])
+        return version
+
+    def get_api_version_list(self, ver):
+        version = list()
+        version.append(ver[:ver.find('.')])
+        ver = ver[ver.find('.') + 1:]
+        version.append(ver[:ver.find('.')])
+        ver = ver[ver.find('.') + 1:]
+        version.append(ver[:ver.find('-')])
+        ver = ver[ver.find('-') + 1:]
+        return version
+
+            
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument('-registry', action='store',
+                        help='Use specified registry file instead of vk.xml')
+    parser.add_argument('-profile_path', action='store',
+                        help='Path to base profile')
+    parser.add_argument('-profile_path2', action='store',
+                        help='Path to second profile')
+    parser.add_argument('-output_path', action='store',
+                        help='Path to output profile')
+    parser.add_argument('-profile', action='store',
+                        help='Base profile to build the new profile from')
+    parser.add_argument('-profile2', action='store',
+                        help='Profile to combine into the base profile')
+    parser.add_argument('-output_profile', action='store',
```

Comment:
I think we need two modes regarding the output profile name: one that is automatically generated so that it's sort of a unique identifier:
Maybe VP_LUNARG_merged_YYYY_MM_DD_HH_MM?

And one where the user set the profile name. We will probably have to check that the user provided name follow some sort of naming convention.  The schema requires: "^VP_[A-Z0-9]+_[A-Za-z0-9_]+"